analyses/TN_cd_2000/02_setup_TN_cd_2000.R

```diff
@@ -0,0 +1,23 @@
+###############################################################################
+# Set up redistricting simulation for `TN_cd_2000`
+# © ALARM Project, July 2025
+###############################################################################
+cli_process_start("Creating {.cls redist_map} object for {.pkg TN_cd_2000}")
+
+map <- redist_map(tn_shp, pop_tol = 0.005,
+    existing_plan = cd_2000, adj = tn_shp$adj)
+
+# Adjust pop_muni as needed to balance county/muni splits
+# make pseudo counties with default settings
+map <- map %>%
+    mutate(pseudo_county = pick_county_muni(map, counties = county, munis = muni,
+        pop_muni = get_target(map)))
+# IF MERGING CORES OR OTHER UNITS:
```

Comment:
Can you remove comments like this that aren't used. This helps us keep the final scripts clean.